### PR TITLE
[ja] Add dockerhim depreaction FAQ blog post

### LINKED ISSUE #39251: [ja] Add translations for dockershim related contents
**This is a Feature Request**

**What would you like to be added**

Add japanese translations of the contents listed below:

- {{% dockershim-removal %}} block
- [Dockershim Removal FAQ](https://kubernetes.io/blog/2022/02/17/dockershim-faq/)
- [Check whether Dockershim removal affects you](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)

**Why is this needed**

**Comments**

ref. #39125

### PR DESCRIPTION
**Pull request by grainrigi**
Add dockerhim depreaction FAQ blog post and modify related links/messages.

- add https://kubernetes.io/blog/2022/02/17/dockershim-faq/
- rename https://kubernetes.io/ja/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/ to check-if-dockershim-**removal**-affects-you and chenge "deprecation" to "removal" in its content (from 9211599)
- add dockershim-removal codeblock i18n (adding dockershim_message variable)

This closes #39251.

### CODE REVIEW

File: content/ja/blog/_posts/2022-02-17-updated-dockershim-faq.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+`cri-dockerd`をインストールして、kubeletをDocker Engineに接続するために使用することができます。
+詳細については、[Migrate Docker Engine nodes from dockershim to cri-dockerd](/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd/)を読んでください。
+
+[mirantis]: https://www.mirantis.com/blog/mirantis-to-take-over-support-of-kubernetes-dockershim-2/
+
+### 今現在でプロダクション環境に他のランタイムを使用している例はあるのでしょうか？
+
+Kubernetesプロジェクトが生み出したすべての成果物(Kubernetesバイナリ)は、リリースごとに検証されています。
+
+また、[kind] プロジェクトは以前からcontainerdを使っており、
```
Comment (nasa9084) [inline content/ja/blog/_posts/2022-02-17-updated-dockershim-faq.md:110]:
```suggestion
また、[kind]プロジェクトは以前からcontainerdを使っており、
```

File: content/ja/blog/_posts/2022-02-17-updated-dockershim-faq.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+プロジェクトのユースケースにおいて安定性が向上してきています。
```
Comment (nasa9084) [inline content/ja/blog/_posts/2022-02-17-updated-dockershim-faq.md:111]:
```suggestion
また、[kind]プロジェクトは以前からcontainerdを使っており、プロジェクトのユースケースにおいて安定性が向上してきています。
```

**Comment by grainrigi** [comment]
@nasa9084 Thank you! I applied your suggestions.

**Comment by nasa9084** [comment]
/approve
please wait for other reviewer to review

File: content/ja/blog/_posts/2022-02-17-updated-dockershim-faq.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+- KubernetesプロジェクトのツールでDocker Engineへの直接アクセスが必要なもの(例:廃止された`kube-imagepuller`ツール)
+- `registry-mirrors`やinsecureレジストリなどの機能の設定
+- その他の支援スクリプトやデーモンでDocker Engineが利用可能であることを想定していてKubernetes外で実行されるもの(モニタリング・セキュリティエージェントなど)
+- GPUまたは特別なハードウェア、そしてランタイムおよびKubernetesとそれらハードウェアの統合方法
+
+あなたがKubernetesのリソース要求/制限やファイルベースのログ収集DaemonSetを使用しているのであれば、それらは問題なく動作し続けますが、
+dockerdの設定をカスタマイズしていた場合は、それを新しいコンテナランタイムに適合させる必要があるでしょう。
+
+他に注意することとしては、システムメンテナンスを実行するようなものや、コンテナ内でイメージをビルドするようなものが動作しなくなります。
+前者の場合は、[crictl][cr]ツールをdrop-inの置き換えとして使用できます([docker cliからcrictlへのマッピング](https://kubernetes.io/ja/docs/tasks/debug/debug-cluster/crictl/#docker-cli%E3%81%8B%E3%82%89crictl%E3%81%B8%E3%81%AE%E3%83%9E%E3%83%83%E3%83%94%E3%83%B3%E3%82%B0)を参照)。
```
Comment (t-inu) [inline content/ja/blog/_posts/2022-02-17-updated-dockershim-faq.md:156]:
```suggestion
前者の場合は、[`crictl`][cr]ツールをdrop-inの置き換えとして使用できます([docker cliからcrictlへのマッピング](https://kubernetes.io/ja/docs/tasks/debug/debug-cluster/crictl/#docker-cli%E3%81%8B%E3%82%89crictl%E3%81%B8%E3%81%AE%E3%83%9E%E3%83%83%E3%83%94%E3%83%B3%E3%82%B0)を参照)。
```
> you can use the [`crictl`][cr] tool as a drop-in replacement

「docker cliからcrictlへのマッピング」へのリンクについては、原文ではPR https://github.com/kubernetes/website/pull/31316 で、このセクション自体がなくなり、リファレンスにページを作ったようです。
https://kubernetes.io/docs/reference/tools/map-crictl-dockercli/
原文から直すのが原則だと思いますので、とりあえずは今のままにしておきましょうか。

File: content/ja/blog/_posts/2022-02-17-updated-dockershim-faq.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+[ドキュメント]: https://github.com/containerd/cri/blob/master/docs/registry.md
+
+containerdとCRI-OをKubernetesで使用する方法に関しては、[コンテナランタイム]に関するKubernetesのドキュメントを参照してください。
+
+[コンテナランタイム]: /ja/docs/setup/production-environment/container-runtimes/
+
+### さらに質問がある場合どうすればいいでしょうか？
+
+ベンダーサポートのKubernetesディストリビューションを使用している場合、彼らの製品に対するアップグレード計画について尋ねることができます。
+エンドユーザーの質問に関しては、[エンドユーザーコミュニティーフォーラム](https://discuss.kubernetes.io/)に投稿してください。
```
Comment (t-inu) [inline content/ja/blog/_posts/2022-02-17-updated-dockershim-faq.md:176]:
```suggestion
エンドユーザーの質問に関しては、[エンドユーザーコミュニティフォーラム](https://discuss.kubernetes.io/)に投稿してください。
```

**Review by t-inu — COMMENTED** [review]
Thank you for translating this important article.
Please check my comments.

**Comment by grainrigi** [comment]
@t-inu Thank you for your thorough review! I applied your suggestions.

**Review by t-inu — APPROVED** [review]
Thanks!
/lgtm

**Comment by t-inu** [comment]
/retitle [ja] Add dockershim deprecation FAQ blog post

**Pull request merged**